README.md

```diff
@@ -20,12 +21,9 @@ npm i
 npm run pack
 ```
 
-## Run Sample Code
+## Run sample code
 
-Once you do this you can use it as the sample shows in [here](samples/sample.js).
-```
-node sample/sample.js
-```
+There is a sample file included [here](samples/sample.ts).
```

Comment:
Command To Run The Sample File?
Maybe The Command To Run The Tests Can Also Be Helpful

Reply:
I prefer not to include any reference to any external library here. I'll let the users decide how to run the sample

Reply:
So what about changing the header?

`## Sample code`

Reply:
Makes sense, thanks